Storage/Storage/Model/CustomerSearchResult+CoreDataProperties.swift

```diff
@@ -8,9 +8,9 @@ extension CustomerSearchResult {
         return NSFetchRequest<CustomerSearchResult>(entityName: "CustomerSearchResult")
     }
 
-    @NSManaged public var customerID: Int64
+    @NSManaged public var siteID: Int64
+    @NSManaged public var keyword: String?
```

Comment:
> Add optional keyword property to CustomerSearchResult

Why optional? I don't think it makes sense to have a search result with a nil keyword...

Reply:
🙇  Made the changes discussed in Slack here: 312e9fd466f620394766d2c56c97d52b679d4480 by changing `keyword` to be non-optional and providing a default value so we're sure that the entity conforms to the new schema when migrating